tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -168,6 +173,18 @@ public void ProcessChooserCommand()
         barWindow?.NavigateTo(typeof(ProcessListPageViewModel));
     }
 
+    public void ManageExternalToolsButton_Click(object sender, RoutedEventArgs e)
+    {
+        // Need to be in a horizontal layout
+        BarOrientation = Orientation.Horizontal;
```

Comment:
Why?

Reply:
Our expanded content is only shown in the horizontal position. I'll update the comment.